### DISCUSSION TITLE
Is there no --out-dir flag for V2 Serve?

### DISCUSSION THREAD

**Discussion by Natetronn** [parcel-bundler/parcel#5126] [General] [answered]
[I see there is `--dist-dir <dir>` for Build](https://v2.parceljs.org/features/cli/#parameters-specific-to-the-non-server-commands-(watch-and-build)) but, I don't see an `--out-dir` flag, like there was on v1. What's the current method for outputting separate serve (dev) and build (prod; dist) directories with V2?

**Comment by mischnic** [comment, marked as answer]
It should be there. The docs are outdated

https://github.com/parcel-bundler/parcel/blob/71b21f17a13f48d14c788717f27bfba39ecacafa/packages/core/parcel/src/cli.js#L57-L81

**Comment by Natetronn** [comment]
Awesome, thanks for the quick reply! 

I can use the `--dist-dir` flag now on both Serve and Build, as it's now a common flag? Yup, worked great.

**Comment by Natetronn** [comment]
Serving to serve directory and building to dist directory (default):

```
  "scripts": {
    "serve": "parcel serve ./src/index.html --dist-dir ./serve",
    "build": "rm -rf ./dist && parcel build ./src/index.html"
  },
```
